.clang-tidy

```diff
@@ -0,0 +1,15 @@
+UseColor: true
+
+# TODO: Discuss if we wanna add some extra checks.
+Checks: >
```

Comment:
Add a comment explaining how this list was chosen (maybe one comment per item, if needed)

Reply:
Added a comment! The list was not chosen by me, I took it from @ArthurBandaryk's PR, so I'm not going to bother writing down comments for now, let's do it after we agree on the list of checks